uint32                              id
float64                             deadline
string                              tasks
mapc_rhbp_ettlinger/TaskBid         bid
string[]                            items
string[]                            agents
